### ISSUE TITLE
Ruby 2.15 install/build fails

### ISSUE THREAD

**Issue by Matt-Yorkley** [openfoodfoundation/ofn-install#344]
We install ruby via the community plugin `zzbet.rbenv`. When it gets to the install task, it fails in Ubuntu 18.

Output:

```
Downloading ruby-2.1.5.tar.bz2...
-> https://cache.ruby-lang.org/pub/ruby/2.1/ruby-2.1.5.tar.bz2
Installing ruby-2.1.5...
WARNING: ruby-2.1.5 is past its end of life and is now unsupported.
It no longer receives bug fixes or critical security updates.
BUILD FAILED (Ubuntu 18.04 using ruby-build 20190320)
Inspect or clean up the working tree at /tmp/ruby-build.20190326160938.15756
Results logged to /tmp/ruby-build.20190326160938.15756.log
Last 10 log lines:
make[1]: *** [ext/openssl/all] Error 2
make[1]: *** Waiting for unfinished jobs....
compiling callback-7.c
compiling callback-8.c
installing default callback libraries
linking shared-object dl/callback.so
make[2]: Leaving directory '/tmp/ruby-build.20190326160938.15756/ruby-2.1.5/ext/dl/callback'
make[1]: Leaving directory '/tmp/ruby-build.20190326160938.15756/ruby-2.1.5'
uncommon.mk:180: recipe for target 'build-ext' failed
```

**Comment by Matt-Yorkley** [comment]
Our version of the `zzbet.rbenv` plugin is 2 years out of date. I'll try updating it and see how it goes on Bionic.

**Comment by Matt-Yorkley** [comment]
Updating it doesn't help. It looks like the build error is related to the `openssl` package. I'll open an issue on the plugin maintainer's repo and investigate a bit more.

**Comment by Matt-Yorkley** [comment]
Seems to be an issue with the openssl package, and it affects general building of older Ruby versions. It was changed from 1.0.x to 1.1.x in Ubuntu 18, and the new version doesn't play nicely with older Ruby.

More info here: https://stackoverflow.com/questions/45755066/cannot-install-ruby-with-rbenv-2-3-1-in-ubuntu-16-04

**Comment by enricostano** [comment]
Should we use a tag for all the Ubuntu 18 related issues? And/or a [project](https://github.com/openfoodfoundation/ofn-install/projects) maybe.

**Comment by Matt-Yorkley** [comment]
Installing the package `libssl1.0-dev` makes the necessary version of `openssl` available for compiling when Ruby 2.1.5 is built, and fixes the issue. It doesn't override the system's main version of `openssl`.

The suggestion came form here: https://github.com/rbenv/ruby-build/wiki#openssl-usrincludeopensslasn1_mach102-error-error-this-file-is-obsolete-please-update-your-software

We can either add it to our package list, or we can pass it to `zbbet.rbenv` by setting this variable somewhere in ofn-install:

```
rbenv_extra_depends:
  - libssl1.0-dev
```

Ruby then compiles correctly. :tada: 

The package `libssl1.0-dev` is not available before Bionic Beaver, so maybe we'll have to conditionally install it if it's available? It's probably simple enough with Ansible, but is it the best way to do this?
